crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -318,7 +318,19 @@ impl ShaderCache {
                 render_device
                     .wgpu_device()
                     .push_error_scope(wgpu::ErrorFilter::Validation);
-                let shader_module = render_device.create_shader_module(module_descriptor);
+
+                // `validate_shader` is evaluated to determine whether or not to perform
```

Comment:
I don't think this comment is useful: it's just explaining what the code below says.